src/QMCWaveFunctions/SpinorSet.h

```diff
@@ -15,6 +15,7 @@
 
 #include "QMCWaveFunctions/SPOSet.h"
 #include <ResourceHandle.h>
+#include <sys/wait.h>
```

Comment:
Should not include this.

Reply:
Not sure how that got added either. Thanks for catching this. I think my neovim setup may have added this when I accidentally typed something and triggered some snippet to be added. Sometimes this is useful, and sometimes it isn't. I'll be more careful, and try to make sure my setup isn't stuff like this.

Reply:
fixed